includes/admin/class-sensei-learners-admin-bulk-actions-view.php

```diff
@@ -170,7 +179,7 @@ protected function get_row_data( $item ) {
 			);
 		} else {
 			$learner            = $item;
-			$courses            = $this->get_learner_courses_html( $item->course_statuses );
```

Comment:
As we are not using the course_statuses field anymore, can we stop fetching it from the database in the Sensei_Db_Query_Learners::build_query function or make it optional? As a group concat operation is performed to prepare the data for this column. WDYT? Also if you decide to work on it, can you also take a look at the left join that we do with wp_comment table to see if we need it anymore for this page?

Reply:
The problem is that it is public, so I am not sure if it is okay to remove part of data.
The other reason for me to keep it "as is" was that we will consider displaying statuses of courses (completed or not) soon.

Reply:
@Imran92 Fixed here: https://github.com/Automattic/sensei/pull/5043/commits/4d72a99320f07ccc131446229db2012ee0cb010b